crates/weechat/src/config/config.rs

```diff
@@ -320,7 +321,12 @@ impl Config {
                 weechat_ptr,
                 _phantom: PhantomData,
             }),
-            _ => unreachable!(),
+            "enum" => ConfigOption::Enum(EnumOption {
+                ptr: option_ptr,
+                weechat_ptr,
+                _phantom: PhantomData,
+            }),
+            _ => todo!("Outdated option_from_type_and_ptr"),
```

Comment:
Replaced `unreachable!()` macro with `todo!()` since the first one doesn't allow any message, but I'm thinking it would be better to avoid panicking, maybe returning `Result<_>` would be better, need to check signature usages across lib

Reply:
Yeah, a `Result` might be the best option here.